overlord/snapshotstate/backend/backend.go

```diff
@@ -476,9 +476,69 @@ func addDirToZip(ctx context.Context, snapshot *client.Snapshot, w *zip.Writer,
 
 var ErrCannotCancel = errors.New("cannot cancel: import already finished")
 
+// XXX: move to a new errutil ?
```

Comment:
This would be great; AFAIR there were cases already where we collated multiple errors into one, outside of snapshots.